<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationDialog</class>
 <widget class="QDialog" name="StationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络设备</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>基本设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>站号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="num"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>262</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string> 设置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>MonitorCtrl</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>BAMS</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="devices"/>
          </item>
          <item row="4" column="1" colspan="3">
           <widget class="QPlainTextEdit" name="editTagDesc"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>devices</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>变量描述</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Pump</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="Pump"/>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>PumpCtrl</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="PumpCtrl"/>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>PowerCtrl</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>PCS</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="PCS"/>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="PowerCtrl"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="MonitorCtrl"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="BAMS"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPropertySetting">
      <attribute name="title">
       <string> 设备属性</string>
      </attribute>
      <layout class="QVBoxLayout" name="vLayoutPropertyEditor"/>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="btnHelp">
       <property name="text">
        <string>帮助</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCheck">
       <property name="text">
        <string>检查</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExit">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
